model-archiver/model_archiver/tests/unit_tests/test_model_packaging.py

```diff
@@ -67,32 +58,32 @@ def test_export_model_method(self, patches):
         )
         patches.export_utils.zip.return_value = None
 
-        package_model(self.args, ModelExportUtils.generate_manifest_json(self.args))
+        package_model(self.config, ModelExportUtils.generate_manifest_json(self.config))
         patches.export_utils.validate_inputs.assert_called()
         patches.export_utils.archive.assert_called()
 
     def test_export_model_method_tar(self, patches):
-        self.args.update(archive_format="tar")
+        self.config.archive_format = "tar"
```

Comment:
This value "tar" doesn't align with the list of `choices` originally accepted in the CLI https://github.com/pytorch/serve/blob/master/model-archiver/model_archiver/arg_parser.py#L102. Should we add "tar" to the literal?

Reply:
yes let's add